packages/protocol-kit/tests/e2e/safeFactory.test.ts

```diff
@@ -139,9 +139,9 @@ describe('SafeProxyFactory', () => {
       const threshold = 0
       const safeAccountConfig: SafeAccountConfig = { owners, threshold }
       const safeDeploymentConfig: SafeDeploymentConfig = { saltNonce: '1' }
-      const predictSafeProps: PredictSafeProps = { safeAccountConfig, safeDeploymentConfig }
+      const PredictedSafeProps: PredictedSafeProps = { safeAccountConfig, safeDeploymentConfig }
```

Comment:
The first should start with lower `p` `predictedSafeProps` to follow constant naming convention